react/src/lib/components/SubsurfaceViewer/components/Map.tsx

```diff
@@ -429,20 +429,29 @@ const Map: React.FC<MapProps> = ({
                 viewsProps.map((item) => [item.id, cameraPosition])
             ) as unknown as Record<string, ViewStateType>;
         } else {
+            const isBoundsDefined = typeof bounds !== "undefined";
+
             tempViewStates = Object.fromEntries(
                 viewsProps.map((item, index) => {
                     const viewState = viewStates[item.id];
                     return [
                         item.id,
                         typeof viewState !== "undefined"
                             ? viewState
-                            : getViewState(
+                            : isBoundsDefined
```

Comment:
Nested ternary operators are hard to read.

https://eslint.org/docs/latest/rules/no-nested-ternary

Suggest adding the `no-nested-ternary` condition to the eslint config.

Reply:
Suggest adding lint rule "no-nested-ternary" is separate PR as it requires multiple unrelated files to be rewritten.